com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -183,6 +188,10 @@ out input
             UpdateEnvironmentWithInput(input.RlInput);
             initParametersOut = initializationInput.RlInitializationInput.ToUnityRLInitParameters();
             return true;
+#else
+            initParametersOut = new UnityRLInitParameters();
```

Comment:
does this still reference the grpc stuff?

Reply:
does that matter?

Reply:
No, this is the C#-only class.

The breaking change in the other PR was referencing RpcException and the status codes, the classes in the signature didn't change (although the return value was moved to an `out` parameter)